src/script/components/calling/CallingOverlayContainer.tsx

```diff
@@ -123,6 +123,10 @@ const CallingContainer: React.FC<CallingContainerProps> = ({
     callingRepository.refreshAudioInput();
   };
 
+  const switchSpeakerOutput = (call: Call, deviceId: string) => {
```

Comment:
question: Can we remove call param? It doesn't seem to be used.

Reply:
ive removed this.